src/services/authService.js

```diff
@@ -1,47 +1,42 @@
 import http from "./httpService";
 import { apiUrl } from "../config.json";
-import jwtDecode from "jwt-decode";
+import { toast } from "react-toastify";
 
-const apiEndpoint = `${apiUrl}/auth/login`;
-const tokenKey = "token";
-
-http.setJwt(getJwt());
-
-export async function login(userName, password) {
-  const { data: jwt } = await http.put(apiEndpoint, {
-    userName,
-    password,
-  });
-
-  const jwt2 = jwt["accessToken"];
-  return localStorage.setItem(tokenKey, jwt2);
-}
-
-export function loginWithJwt(jwt) {
-  localStorage.setItem(tokenKey, jwt);
+export async function login(credentials) {
+  return await http.put(`${apiUrl}/auth/login`, credentials);
 }
 
-export function logout() {
-  localStorage.removeItem(tokenKey);
+export async function refresh(accessToken, refreshToken) {
+  const request = {
+    accessToken,
+    refreshToken,
+  };
+  try {
+    return await http.put(`${apiUrl}/auth/refresh`, request);
+  } catch (error) {
+    toast.error(error);
+  }
 }
 
-export function getCurrentUser() {
+export function registerUser(user) {
   try {
-    const jwt = localStorage.getItem(tokenKey);
-    return jwtDecode(jwt);
-  } catch (ex) {
-    return null;
+    return http.post(`${apiUrl}/auth/register/user`, user);
+  } catch (error) {
+    toast.error(error);
   }
 }
 
-export function getJwt() {
-  return localStorage.getItem(tokenKey);
+export function registerShelter(shelter) {
+  try {
+    return http.post(`${apiUrl}/auth/register/shelter`, shelter);
```

Comment:
Myslim, ze nieco som uz pisal aj v nejakom backend PR. Nepride mi spravne (rozumej logicke) mat registraciu shelter zavesenu na auth endpoint.

Reply:
ak to stihnem este dnes tak to urobim kazdopadne mam to v poznamkach TO DO